24H Study Room/Models/Email.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Email.swift
+//  24H Study Room
+//
+//  Created by Jia-Jiuan Tsai on 2022-09-06.
+//
+
+import Foundation
+
+struct Email {
+    let emailString: String
+    
+    init?(emailString: String) {
```

Comment:
This constructor is missing tests. 

There does not appear to be any tests at all in this project so far. There are different philosophies for exactly how much test coverage is needed and in what situations. But code like this, that does complex checks or transformations of data, definitely needs tests in my opinion. And tests are not something to add later – they should be a constant part of development